src/scenic/core/regions.py

```diff
@@ -2055,6 +2062,152 @@ def evaluateInner(self, context):
         )
 
 
+class VoxelRegion(Region):
+    """Region represented by a voxel grid in 3D space.
+
+    Args:
+        encoding: A numpy array encoding the voxel grid.
+        dimensions: An optional 3-tuple, with the values representing width, length, height
+          respectively. The voxel region will be scaled to have these dimensions. Default
+          value (1,1,1).
+        position: A position, which determines where the center of the region will be. Default
```

Comment:
These arguments don't appear in the signature: it seems `voxelGrid` must be used.

Reply:
Ah good catch, the docstring is out of date. I'll update it.